Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/KotlinCodeModel.kt

```diff
@@ -339,9 +339,9 @@ abstract class KotlinFile(
 internal fun getTopLevelPropertyDeclarationName(scope: KotlinScope, property: PropertyStub): String {
     val receiverName = when (val type = property.receiverType) {
         null -> null
-        is ClassifierStubType -> type.classifier.getRelativeFqName()
-        is AbbreviatedType -> type.abbreviatedClassifier.getRelativeFqName()
-        is FunctionalType -> type.classifier.getRelativeFqName()
+        is ClassifierStubType -> type.classifier.fqName
+        is AbbreviatedType -> type.abbreviatedClassifier.fqName
```

Comment:
This doesn't seem consistent with Kotlin overloading rules.
Maybe use `.underlyingType`?